yt_dlp/extractor/bitchute.py

```diff
@@ -74,11 +86,19 @@ def _check_format(self, video_url, video_id):
                 'filesize': int_or_none(response.headers.get('Content-Length'))
             }
 
+    @staticmethod
+    def _raise_if_restricted(webpage):
+        page_title = clean_html(get_element_by_class('page-title', webpage))
+        if page_title and re.fullmatch(r'(?:Channel|Video) Restricted', page_title):
+            reason = clean_html(get_element_by_id('page-detail', webpage)) or page_title
+            raise GeoRestrictedError(reason)
```

Comment:
use `self.raise_geo_restricted`

You may also want to set `_GEO_BYPASS = False`